pymatgen/analysis/quasirrho.py

```diff
@@ -0,0 +1,219 @@
+# Distributed under the terms of the MIT License.
+
+
+"""
+A module to calculate free energies using the Quasi-Rigid Rotor Harmonic
+ Oscillator approximation. Modified from a script by Steven Wheeler.
+See: Grimme, S. Chem. Eur. J. 2012, 18, 9955
+"""
+
+__author__ = "Alex Epstein"
+__copyright__ = "Copyright 2020, The Materials Project"
+__version__ = "0.1"
+__maintainer__ = "Alex Epstein"
+__email__ = "aepstein@lbl.gov"
+__date__ = "January 5, 2021"
+__credits__ = "Steven Wheeler, Trevor Seguin, Evan Spotte-Smith"
+
+from math import isclose
+from typing import Union
+
+import numpy as np
+import scipy.constants as const
+
+from pymatgen.core.units import amu_to_kg
+from pymatgen.core.units import kb as kb_ev
+from pymatgen.io.gaussian import GaussianOutput
+from pymatgen.io.qchem.outputs import QCOutput
+
+# Define useful constants
+kb = kb_ev * const.eV  # pymatgen kb from ev/K to J/K
+c = const.speed_of_light * 100  # cm/s
+h = const.h  # Planck's constant J.s
+R = const.R / const.calorie  # Ideal gas constant cal/mol
+
+# Define useful conversion factors
+kcal2hartree = 0.0015936  # kcal/mol to hartree/mol
+
+
+def get_avg_mom_inertia(mol):
+    """
+    Caclulate the average moment of inertia of a molecule
+    :param mol: Molecule
+    :return: average moment of inertia, eigenvalues of inertia tensor
+    """
+    centered_mol = mol.get_centered_molecule()
+    inertia_tensor = np.zeros((3, 3))
+    for site in centered_mol:
+        c = site.coords
+        wt = site.specie.atomic_mass
+        for i in range(3):
+            inertia_tensor[i, i] += wt * (c[(i + 1) % 3] ** 2 + c[(i + 2) % 3] ** 2)
+        for i, j in [(0, 1), (1, 2), (0, 2)]:
+            inertia_tensor[i, j] += -wt * c[i] * c[j]
+            inertia_tensor[j, i] += -wt * c[j] * c[i]
+
+    inertia_eigenvals = np.multiply(np.linalg.eig(inertia_tensor)[0], amu_to_kg * 1e-20).tolist()  # amuangs^2 to kg m^2
+
+    iav = np.average(inertia_eigenvals)
+
+    return iav, inertia_eigenvals
+
+
+class QuasiRRHO:
+    """
+    Class to calculate thermochemistry using Grimme's Quasi-RRHO approximation.
+    All outputs are in atomic units.
+    """
+
+    def __init__(
+        self, output: Union[GaussianOutput, QCOutput, dict], sigma_r=1, temp=298.15, press=101317, conc=1, v0=100
+    ):
+        """
+
+        :param output: Requires input of a Gaussian output file,
+                        QChem output file, or dictionary of necessary inputs:
+                        {"mol": Molecule, "mult": spin multiplicity (int),
+                        "frequencies": list of vibrational frequencies [a.u.],
+                        elec_energy": electronic energy [a.u.]}
+        :param sigma_r (int): Rotational symmetry number
+        :param temp (float): Temperature [K]
+        :param press (float): Pressure [Pa]
+        :param conc (float): Solvent concentration [M]
+        :param v0 (float): Cutoff frequency for Quasi-RRHO method [cm^1]
```

Comment:
If possible, please convert to a [google style docstring](https://gist.github.com/redlotus/3bc387c2591e3e908c9b63b97b11d24e) 

Since all of the outputs are stored as class attributes, it would also be especially nice to have a `Attributes` section that describes each output, with units

Reply:
Alternatively, you could define python `@property` methods for each of the relevant outputs (e.g., `g_corrected`, h_corrected`, which would let you give each one an explicit docstring. Each property would not do anything other than contain the docstring, e.g.

```
@property
def g_corrected(self):
'''
Corrected free energy in Ha
'''
return self.g_corrected
```

so the only benefit to this structure would be visibility of the outputs to users